backend/baseclass.pm

```diff
@@ -404,6 +407,22 @@ sub cpu_stat {
     return [];
 }
 
+sub format_vtt_timestamp {
+    my ($self, $walltime) = @_;
+
+    my $frametime_ms = 1000 * $self->{video_frame_number} / 24;
+    my $caption = "\n$self->{video_frame_number}\n";
+    # presentation time span (one frame)
+    $caption .= sprintf(POSIX::strftime("%T.%%03d", gmtime($frametime_ms / 1000)), $frametime_ms % 1000);
```

Comment:
do we really need this subsecond? I would say no.

Reply:
I prefer to have it the same format as the timestamp in autoinst.log

Reply:
Yes, I would appreciate the subsecond precision